tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -663,8 +663,8 @@ def _check_ssh_key(user, ssh_generation_enabled, remote_command_executor, schedu
     [
         ("SimpleAD", "ldap", False),
         # ("SimpleAD", "ldaps", False),
-        ("SimpleAD", "ldaps", True),
-        ("MicrosoftAD", "ldap", False),
+        # ("SimpleAD", "ldaps", True),
```

Comment:
should we just delete the tests instead of leaving a comment?

Reply:
I understand your point, but I prefer not to lose the track of the previous coverage